README.md

```diff
@@ -15,26 +15,28 @@ Setup for developers (Unix)
 ---------------------------
 
 1. Make sure you have installed Python 3.6, [pip3](https://pip.pypa.io/en/latest/) and [virtualenv](http://www.virtualenv.org/en/latest/).
-1. If working behind a proxy, make sure your environment variables are properly set up. If
+2. If working behind a proxy, make sure your environment variables are properly set up. If
    you still get an error due to proxy, use "-E" flag along with "sudo" to export all the
    environment variables.
-1. Make sure you have python3-dev installed on your operating system. For Debian, you would additionally require libpq-dev.
+3. Make sure you have python3-dev installed on your operating system. For Debian, you would additionally require libpq-dev.
    Install by using `sudo apt-get install libpq-dev python3-dev`
-1. Make sure you have PostgreSQL installed. For a tutorial on installing
+4. Make sure you have PostgreSQL installed. For a tutorial on installing
    Postgres, [Django Girls'](http://djangogirls.org) ebook,
    [Tutorials Extension](http://djangogirls.org/resources/), is a reference.
    The info is also on [Django Girls GitHub repository](https://github.com/DjangoGirls/tutorial-extensions/blob/master/en/optional_postgresql_installation/README.md).
-1. Clone the repo - `git clone https://github.com/systers/portal.git` and cd into
+5. Clone the repo - `git clone https://github.com/systers/portal.git` and cd into
   the `portal` directory. If working behind a proxy, follow the instructions [here](https://cms-sw.github.io/tutorial-proxy.html).
-1. Create a virtual environment with Python 3 and install dependencies:
+6. Create a virtual environment with Python 3.6 and install dependencies:
 
      ```bash
+     $ virtualenv -p python3.6 venv
+     `incase of path error use the following command`
      $ virtualenv venv --python=/path/to/python3
```

Comment:
The issue mentions that we should provide a way to find /path/to/python3. Can you work on that? @LaibaBasit008

Reply:
I have made the desired changes